Source/WebCore/dom/Document.cpp

```diff
@@ -4995,6 +4995,11 @@ void Document::setCSSTarget(Element* newTarget)
     m_cssTarget = newTarget;
 }
 
+Element* Document::cssTarget() const
```

Comment:
Was the moving out of line really needed?

Reply:
Yes. `WeakPtr::get()` requires the full type information of `Element`.